web/themes/custom/recipe_theme/src/sass/content/_node.landing-page.scss

```diff
@@ -3,3 +3,16 @@
   margin-bottom: 3rem;
   opacity: 0.5;
 }
+
+.landing-page__body {
+  text-align: center;
+  margin-left: 6rem;
+  margin-right: 6rem;
+}
+
+@media (max-width: 599px) {
```

Comment:
Let's move media queries to the next level. There is a very good article about [how to do breakpoints correctly](https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/).

Also, as we are using SASS you can utilize one of its best functionalities, mixins. This article proved the exact tools you need to use breakpoints with mixins.

Another thing to note: when styling elements, it's a good practice to write css with a 'mobile first' approach, meaning that the default style (the one without media queries) is the one that works for mobile. Then you can begin using mixins for larger screens (min-width 600px, min-width 900px etc.) to adjust the styling for larger screens.

Reply:
I've added some modifications. However, I chose not to use mixins here, because my teammate Mari will be doing site-wide theming after the rest of us finish theming the individual pages separately.

Reply:
I suggest you create the mixins anyway, because I'd suggest Mari to use the mixins as well, you can help her work if you add them now. :slightly_smiling_face: 

There's an [existing empty mixins file in your theme](https://github.com/Pronovix/food/blob/master/web/themes/custom/recipe_theme/src/sass/base/_mixins.scss), you can use this file to put the mixins in.

And to make this code part look better, you don't need to redefine the class every time, you can also add the mixins or media queries inside the class. E.g.:

```sass
.landing-page__body {
  default rules...

  @include mixin1 {
    specific rules...
  }

  @media (min-width: 600px) {
    other specific rules...
  }
}
```